src_zlib/service/zlib_service.cpp

```diff
@@ -335,7 +335,7 @@ init(bool use_gzip)
     stream_.avail_in = 0;
 }
 
-filter::results
+http_proto::detail::filter::results
```

Comment:
auto with a trailing return type might make this shorter?

Reply:
`deflate_filter` is in a different namespace. we need the whole `http_proto::detail::filter::results`.